### PR TITLE
Milestone #5 - Display Joined Missions and Reserved Rockets in My Profile

### LINKED ISSUE #2: [2pt]  Display joined missions in My profile - Filters
Render a list of all joined missions (use `filter()`) on the "My profile" page.

### LINKED ISSUE #3: [2pt]  Display reserved rockets in My profile - Filters
Render a list of all reserved rockets (use `filter()`) on the "My profile" page.

### PR DESCRIPTION
**Pull request by Anjacodes**
In this milestone, we worked on: 

* #3 - Render a list of all reserved rockets on the "My profile" page.
* #2 - Render a list of all joined missions on the "My profile" page.
* Added Styling using Bootstrap CSS

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,57 @@
...
+
+React.js
+Redux
 
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-## Available Scripts
+<!-- <h2>Live Demo</h2>
+
+<a href="https://affectionate-mirzakhani-76e566.netlify.app/" rel="noopener noreferrer">Live Demo Link</a> -->
```
Comment (bimbolabuari) [inline README.md]: Let's remove other project information and description from the readme file to make it look cleaner and more professional ✊

File: README.md [hunk 2]
```diff
@@ -1,10 +1,57 @@
...
 
-In the project directory, you can run:
+For this project you don't need any aditional tools to run it.
+Be sure to have git set up on your local machine. If you are unfamiliar with the necessary steps, please find a step-by-step guide <a href="https://git-scm.com/book/en/v2/Getting-Started-First-Time-Git-Setup" rel="noopener noreferrer">here</a>
+
+<h2>Setup</h2>
+
+Fork a copy to your repository and clone it locally: `
+ $ git clone git@github.com:vicmaburrito/Space-Travelers-Hub.git
+$ cd bookstore `
```
Comment (bimbolabuari) [inline README.md]: Let's remove other project information and description from the readme file to make it look cleaner and more professional ✊

Kindly check for other occurrences 🙂

File: src/components/MyProfile/MyProfile.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+            ))}
+          </ul>
+        ) : (
+          <ul className="data-list py-3 padLeft-0">
+            <div className="data-list-container d-flex justify-content-start align-items-center">
+              <li className="padLeftBottom">No reservations</li>
+            </div>
+          </ul>
+        )}
+      </div>
```
Comment (bimbolabuari) [inline src/components/MyProfile/MyProfile.js:52]: Let's add the `reserved` button to indicate reserved rockets as it's in the required wireframe 👍 

> Required Rocket wireframe
Screenshot (381)


> Your Rocket Screenshot
Screenshot (380)

**Review by bimbolabuari — CHANGES_REQUESTED** [review]
Hi Team, 

You have done a good job so far 🚀  

## Highlights 🎉 

- Install React-Redux, Redux Logger, and React Router.
- `useSelector()` is used to get a state slice corresponding to routes
- No Linter error
- The My Profile section displays all reserved rockets and space missions.
- The Missions section displays a list of current missions along with their brief description and participation status. There is also a button next to each mission that allows users to join the selected mission or leave the mission the user joined earlier
- The Rockets section displays a list of all available SpaceX rockets. Users can book each rocket by clicking the reservation button or cancel the previously made booking

There are some issues that you still need to work on to go to the next project but you are almost there! 🎉 

## Required Changes ♻️
- [x] As required, Kindly add unit tests with React Testing Library and Jest 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

--------


_Reviewed by  @bimbolabuari_

File: README.md (hunk 1 above)
Comment (Anjacodes) [reply README.md]: Hi @bimbolabuari, thank you for your feedback and for being attentive. All your suggestions have been implemented! 🙏

**Review by er-contreras — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

- Well done!

Felix

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**